catalogue/webapp/pages/search/images.tsx

```diff
@@ -124,23 +126,25 @@ const ImagesSearchPage: NextPageWithLayout<Props> = ({
       </Head>
 
       <div className="container">
-        <SearchFilters
-          query={queryString}
-          linkResolver={linkResolver}
-          searchFormId="searchPageForm"
-          changeHandler={() => {
-            const form = document.getElementById('searchPageForm');
-            form &&
-              form.dispatchEvent(
-                new window.Event('submit', {
-                  cancelable: true,
-                  bubbles: true,
-                })
-              );
-          }}
-          filters={filters}
-          isNewStyle
-        />
+        {isEnhanced && (
```

Comment:
Could we include the container div in the condition to avoid empty div?

Reply:
solved